src/walletConnect/screens/ActionRequest.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import 'react-native'
+import { Provider } from 'react-redux'
+import { Screens } from 'src/navigator/Screens'
+import { SupportedActions } from 'src/walletConnect/constants'
+import ActionRequest from 'src/walletConnect/screens/ActionRequest'
+import { Actions } from 'src/walletConnect/v1/actions'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+
+describe('ActionRequest', () => {
+  const store = createMockStore({})
+
+  beforeEach(() => {
+    store.dispatch = jest.fn()
```

Comment:
nit: this works but you can also avoid it and then use `store.getActions()`. Example: https://github.com/valora-inc/wallet/blob/2b28b691640c52bdeeb1b49a7751c5bdf3b764fe/src/exchange/WithdrawCeloReviewScreen.test.tsx#L41-L48